tests/test_charge_point_v16.py

```diff
@@ -821,7 +823,7 @@ async def send_meter_periodic_data(self):
                             "value": "89.00",
                             "context": "Sample.Periodic",
                             "measurand": "Power.Reactive.Import",
-                            "unit": "W",
+                            "unit": "var",
```

Comment:
@N3rdix looks good to me, you could remove this unit line which would confirm the default unit of var is being assigned by the integration.

Reply:
When I do that the asserts both fail but I could not yet find the reason yet (must be test related as the integration itself creates the HA entity correctly now in the devcontainer) 😕

Reply:
Ahhh, that's because the metric class is not actually the sensor but a representation of the raw data read from the charger. To get the sensor value or unit for the assert it'd be similar to line 14 in this https://github.com/MatthewFlamm/pytest-homeassistant-custom-component/blob/master/tests/test_sensor.py

Reply:
That added the missing piece to my head 😄 I've added a separate `test_sensor.py`, I think this might be helpful for future sensor tests anyways

Reply:
Just be aware that in order to test/assert in a sensor how data is processed you'd need to set the metric value/unit in the central system first as there is no dummy charger injecting data.